stories/6-Home.stories.js

```diff
@@ -26,7 +26,7 @@ export const Hero = () => (
           <SubtleLink href="#">Login</SubtleLink>
         </li>
       </ul>
-      <img src={logo} />
+      <img src={logo} alt="open-sauced Logo" />
```

Comment:
Thanks very much for the contribution! Everything looks good, just found this tidbit where the alt text should probably be "Open Sauced logo" instead of "open-sauced Logo"
```suggestion
      <img src={logo} alt="Open Sauced logo" />
```